programs/compile/webpack.config.js

```diff
@@ -135,7 +135,16 @@ function createWebpackConfig(args = [], opts = {}) {
         filename: opts.outputCss || "styles.css",
         allChunks: true
       })
-    ]
+    ],
+    devServer: {
+      historyApiFallback: true,
```

Comment:
As can be seen I've set default value of historyApiFallback to true. This is needed when working with HMR and browser-only routing.

E.g. if you go to `kitsune.dev:8000`, then webpack-dev-server will serve you the `index.html` (and it's `.js` file). If you then are routed to `kitsune.dev:8000/something/else` and then do a full reload of the page then webpack is trying to serve a file at `/something/else` which isn't available. With historyApiFallback set to true it'll always serve the right index file.

Might can be a breaking change. E.g. Kitsune frontend wants some of the requests to go to the Kitsune backend. To solve this you've to add a `proxy` section to the `javascript-env.js` file, like this:

```
devServer: {
  proxy: {
    '/api/**': {
      target: 'http://0.0.0.0:3100',
      secure: false
    }
  }
```